test/framework/pylint/__init__.py

```diff
@@ -0,0 +1,8 @@
+from test.framework.pylint.clusterrunner_raw_checker import ClusterRunnerRawChecker
+
+
+def register(linter):
+    """
+    Register custom lint checkers with pylint. Any new checkers should also be registered here.
+    """
+    linter.register_checker(ClusterRunnerRawChecker(linter))
```

Comment:
The reason for putting this register call in \_\_init.py\_\_ is so that if we add more checker classes, we'll only need to add another `register_checker` call in this file and won't need to change Makefile.